### PR TITLE
[ENH, MRG] Add provenance to TFR baseline application

### PR DESCRIPTION
**Pull request by alexrockhill**
Part of https://github.com/mne-tools/mne-python/pull/10920.

### CODE REVIEW

**Comment by alexrockhill** [comment]
Ok to merge? Maybe @drammock you could take a look if you have a minute

File: mne/time_frequency/tfr.py [hunk 1]
```diff
@@ -904,6 +904,8 @@ def tfr_multitaper(inst, freqs, n_cycles, time_bandwidth=4.0,
 class _BaseTFR(ContainsMixin, UpdateChannelsMixin, SizeMixin, TimeMixin):
     """Base TFR class."""
 
+    baseline = None
```
Comment (agramfort) [inline mne/time_frequency/tfr.py:907]: I would put this in the init

and I would update a test

**Comment by alexrockhill** [comment]
Ok to merge?

File: mne/time_frequency/tfr.py [hunk 2]
```diff
@@ -904,6 +904,10 @@ def tfr_multitaper(inst, freqs, n_cycles, time_bandwidth=4.0,
 class _BaseTFR(ContainsMixin, UpdateChannelsMixin, SizeMixin, TimeMixin):
     """Base TFR class."""
 
+    def __init__(self):
+        super().__init__()
```
Comment (drammock) [inline mne/time_frequency/tfr.py:908]: none of the mixins have `__init__` methods (right?), so why is it necessary to call `super().__init__()` here?

**Review by drammock — COMMENTED** [review]
changes look reasonable, just one code question (below), and one other question:

> Ok to merge?

are you in a particular hurry to see this merged / is it important to get this into 1.1?

File: mne/time_frequency/tfr.py (hunk 2 above)
Comment (alexrockhill) [reply mne/time_frequency/tfr.py:908]: Good point, it isn't.

**Comment by alexrockhill** [comment]
> changes look reasonable, just one code question (below), and one other question:
> 
> > Ok to merge?
> 
> are you in a particular hurry to see this merged / is it important to get this into 1.1?

Not in a terrible hurry but another PR depends on it which is making progress in GSoC which would be nice to keep things moving forward.

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
@alexrockhill merge when it’s all green 🙏

**Comment by drammock** [comment]
@agramfort "merge when green" is now a clickable button "enable auto-merge (squash)" :)

**Pull request merged**